index.rst

```diff
@@ -446,148 +374,84 @@ requested time, which will change as the telescope track the object.
-        targetName=target_name,
-        targetInstance=ATPtg.TargetInstances.CURRENT,
-        frame=ATPtg.CoordFrame.ICRS,
-        epoch=2000,  # should be ignored: no parallax or proper motion
-        equinox=2000,  # should be ignored for ICRS
-        ra=radec.ra.hour,
-        declination=radec.dec.deg,
-        parallax=0,
-        pmRA=0,
-        pmDec=0,
-        rv=0,
-        dRA=0,
-        dDec=0,
-        rotPA=180.-alt_az.alt.deg,
-        rotFrame=ATPtg.RotFrame.FIXED,
-        rotMode=ATPtg.RotMode.FIELD,
-        timeout=10
-    )
-
-In case you need to stop tracking, use the next cell!
+As with the calibration tasks, one can use the `take_object` and `take_engtest`
+tasks to get object and engineering test data respectively. These tasks are
+similar to that of `take_flats` where the user can specify a filter and grating
+in addition to an exposure time and number of exposures, e.g.;
 
 ::
 
-    await atptg.cmd_stopTracking.start(timeout=10)
+    object_data_id_list = await latiss.take_object(exptime=5., n=10,
+                                                   filter="blank_bk7_wg05",
+                                                   grating="ronchi90lpmm",
+                                                   object="Alf Vel")
 
-Use the next cell in case you need to offset to center the target on the
-FoV.
+    engtst_data_id_list = await latiss.take_engtest(exptime=5., n=10,
+                                                    filter="blank_bk7_wg05",
+                                                    grating="empty_1",
+                                                    object="Alf Vel - test")
 
-This will set total offsets. So, if you say ``el=0`` and ``az=-30`` and
-then later you do ``el=30`` and ``az=0.``, it will reset the offset in
-azimuth to zero and make an offset of 30arcs in elevation.
+Again, the method will return a list of data ids that can be used to access
+the data on the butler.
 
 ::
 
-    await atptg.cmd_offsetAzEl.set_start(el=0.,
-                                         az=-100.,
-                                             num=0)
+    from lsst.ip.isr.isrTask import IsrTask
 
-If you want to make persistent offsets you can use the following method.
+    import lsst.daf.persistence as dafPersist
 
-::
+    dataPath = '/mnt/dmcs/oods_butler_repo/repo/'
+    butler = dafPersist.Butler(dataPath)
 
-    await atptg.cmd_offsetAzEl.set_start(el=0.,
-                                         az=-100.,
-                                         num=1)
-
-If you want to add your offset to a pointing model file, do the
-following.
-
-::
-
-    await atptg.cmd_pointNewFile.start()
-    await asyncio.sleep(1.)
-    await atptg.cmd_pointAddData.start()
-    await asyncio.sleep(1.)
-    await atptg.cmd_pointCloseFile.start()
+    data_ref = butler.dataRef('raw', **dict(visit=object_data_id_list[0]))
 
+    exposure = isrTask.runDataRef(data_ref).exposure
 
 
 .. _liveview:
```

Comment:
You have to mention somewhere here how to change the M3 position :-) and which port the instrument is on.